packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -118,7 +117,7 @@ const CFDDashboard = props => {
     };
 
     const updateActiveIndex = index => {
-        if (!index) return;
+        if (index === undefined) return;
```

Comment:
we can’t use **!index**, because when function will pass **‘0’** argument ( argument with "0" is demo tab of mt5 dashboard), then index, which equals **“0”** can’t get right condition and handles:        
 **else if (index === 0){
            updated_state.is_demo_tab = false;**
        }